upi/openstack/security-groups.yaml

```diff
@@ -162,15 +190,15 @@
     openstack.cloud.security_group_rule:
       security_group: "{{ os_sg_master }}"
       protocol: tcp
-      remote_ip_prefix: "{{ os_subnet_range }}"
```

Comment:
are these changes were there in our previous patches ?

Reply:
Yes.